Failed to install GUI

## Description

Hi, I'm trying to install GUI but I got this error message:
```
# github.com/zarbchain/zarb-go/cmd/gtk
/usr/local/go/pkg/tool/darwin_amd64/link: running clang failed: exit status 1
ld: warning: ignoring file /opt/homebrew/opt/freetype/lib/libfreetype.dylib, building for macOS-x86_64 but attempting to link with file built for macOS-arm64
Undefined symbols for architecture x86_64:
  "_FcConfigAppFontAddFile", referenced from:
      __cgo_d0cf9f98169b_Cfunc_addFont in 000171.o
  "_FcConfigGetCurrent", referenced from:
      __cgo_d0cf9f98169b_Cfunc_addFont in 000171.o
  "_cairo_arc", referenced from:
      __cgo_b386fe3591b9_Cfunc_cairo_arc in 000152.o
```

### How To Reproduce

`make build_gui`


### Environment

- **Operation System:  macOS m2

--- a/.github/releasers/macos/gui.bundle
+++ b/.github/releasers/macos/gui.bundle
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <app-bundle>
   <meta>
-    <prefix name="default">/usr/local</prefix>
+    <prefix name="default">${env:PREFIX}</prefix>
     <prefix name="bundle">${env:GUI_BUNDLE}</prefix>
     <destination overwrite="yes">${env:ROOT_DIR}</destination>
     <gtk>gtk+-3.0</gtk>

--- a/.github/releasers/releaser_gui_macos.sh
+++ b/.github/releasers/releaser_gui_macos.sh
@@ -10,6 +10,18 @@ BUILD_DIR="${ROOT_DIR}/build"
 PACKAGE_NAME="pactus-gui_${VERSION}"
 PACKAGE_DIR="${ROOT_DIR}/${PACKAGE_NAME}"
 
+# Check if the architecture is amd64 and update FILE_NAME accordingly
+ARCHITECTURE="$(uname -m)" # Get the machine architecture
+
+if [ "${ARCHITECTURE}" = "x86_64" ]; then
+    FILE_NAME="${PACKAGE_NAME}_darwin_amd64"
+elif [ "${ARCHITECTURE}" = "arm64" ]; then
+    FILE_NAME="${PACKAGE_NAME}_darwin_arm64"
+else
+    echo "Unsupported architecture: ${ARCHITECTURE}"
+    exit 1
+fi
+
 mkdir ${PACKAGE_DIR}
 
 echo "Building the binaries"
@@ -56,7 +68,7 @@ echo "Creating dmg"
 # https://github.com/create-dmg/create-dmg
 create-dmg \
   --volname "Pactus GUI" \
-  "${PACKAGE_NAME}_darwin_amd64.dmg" \
+  "${FILE_NAME}.dmg" \
   "${ROOT_DIR}/pactus-gui.app"
 
 echo "Creating archive"
@@ -64,4 +76,4 @@ cp ${BUILD_DIR}/pactus-daemon     ${PACKAGE_DIR}
 cp ${BUILD_DIR}/pactus-wallet     ${PACKAGE_DIR}
 cp -R ${ROOT_DIR}/pactus-gui.app  ${PACKAGE_DIR}
 
-tar -czvf ${ROOT_DIR}/${PACKAGE_NAME}_darwin_amd64.tar.gz -p ${PACKAGE_NAME}
+tar -czvf ${ROOT_DIR}/${FILE_NAME}.tar.gz -p ${PACKAGE_NAME}

--- a/.github/releasers/releaser_gui_macos.sh
+++ b/.github/releasers/releaser_gui_macos.sh
@@ -10,15 +10,15 @@ BUILD_DIR="${ROOT_DIR}/build"
 PACKAGE_NAME="pactus-gui_${VERSION}"
 PACKAGE_DIR="${ROOT_DIR}/${PACKAGE_NAME}"
 
-# Check if the architecture is amd64 and update FILE_NAME accordingly
-ARCHITECTURE="$(uname -m)" # Get the machine architecture
+# Check the architecture
+ARC="$(uname -m)"
 
-if [ "${ARCHITECTURE}" = "x86_64" ]; then
+if [ "${ARC}" = "x86_64" ]; then
     FILE_NAME="${PACKAGE_NAME}_darwin_amd64"
-elif [ "${ARCHITECTURE}" = "arm64" ]; then
+elif [ "${ARC}" = "arm64" ]; then
     FILE_NAME="${PACKAGE_NAME}_darwin_arm64"
 else
-    echo "Unsupported architecture: ${ARCHITECTURE}"
+    echo "Unsupported architecture: ${ARC}"
     exit 1
 fi
 

--- a/.github/workflows/releaser.yml
+++ b/.github/workflows/releaser.yml
@@ -93,6 +93,8 @@ jobs:
   ########################################
   build-gui-macos:
     runs-on: macos-latest
+    env:
+      PREFIX: /usr/local
 
     outputs:
       checksums: ${{ steps.calc_checksums.outputs.checksums }}
@@ -101,7 +103,7 @@ jobs:
     - uses: actions/checkout@v3
 
     - name: Install Dependencies
-      run: brew install gtk+3 librsvg create-dmg coreutils
+      run: brew install gtk+3 librsvg create-dmg coreutils pkg-config gdk-pixbuf
 
     - name: Install Go
       uses: actions/setup-go@v3
@@ -128,6 +130,59 @@ jobs:
           pactus-*.tar.gz
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  ########################################
+  ## Follow these steps to release macOS ARM:
+  ##
+  ## 1- Login to server using VNC (Read the related issue: https://stackoverflow.com/a/9706088)
+  ## 2- Login to the server using SSH
+  ## 3- Install Homebrew: https://brew.sh/
+  ## 4- Add self-runner: https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/adding-self-hosted-runners
+  ## 5- Run the action
+  ##
+  build-gui-macos-arm:
+    runs-on: self-hosted
+    env:
+      # In M1 Macs `/usr/local` moved to `/opt/homebrew`.
+      #  More info: https://earthly.dev/blog/homebrew-on-m1/
+      #
+      PREFIX: /opt/homebrew
+
+    outputs:
+      checksums: ${{ steps.calc_checksums.outputs.checksums }}
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install Dependencies
+      run: brew install gtk+3 librsvg create-dmg coreutils pkg-config gdk-pixbuf
+
+    - name: Install Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.21
+
+    - name: Create release files
+      run: bash ./.github/releasers/releaser_gui_macos.sh
+
+    - name: Calculate sha256sum
+      id: calc_checksums
+      run: |
+        set -e
+        EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+        echo "checksums<<$EOF"                          >> "$GITHUB_OUTPUT"
+        echo "$(sha256sum pactus-*.dmg pactus-*tar.gz)" >> "$GITHUB_OUTPUT"
+        echo "$EOF"                                     >> "$GITHUB_OUTPUT"
+
+    - name: Publish
+      uses: softprops/action-gh-release@v1
+      with:
+        files: |
+          pactus-*.dmg
+          pactus-*.tar.gz
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
 
   ########################################
   build-gui-windows:

--- a/.github/workflows/releaser.yml
+++ b/.github/workflows/releaser.yml
@@ -143,7 +143,7 @@ jobs:
   build-gui-macos-arm:
     runs-on: self-hosted
     env:
-      # In M1 Macs `/usr/local` moved to `/opt/homebrew`.
+      # In M1 `/usr/local` moved to `/opt/homebrew`.
       #  More info: https://earthly.dev/blog/homebrew-on-m1/
       #
       PREFIX: /opt/homebrew

--- a/.github/workflows/releaser.yml
+++ b/.github/workflows/releaser.yml
@@ -136,7 +136,7 @@ jobs:
   ##
   ## 1- Login to server using VNC (Read the related issue: https://stackoverflow.com/a/9706088)
   ## 2- Login to the server using SSH
-  ## 3- Install Homebrew: https://brew.sh/
+  ## 3- Install Homebrew: https://brew.sh/ and add `brew` to PATH
   ## 4- Add self-runner: https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/adding-self-hosted-runners
   ## 5- Run the action
   ##

